src/main/java/com/blackducksoftware/integration/hub/alert/channel/ChannelDescriptor.java

```diff
@@ -35,8 +39,14 @@
 
     public <E extends DatabaseEntity> Class<E> getDistributionEntityClass();
 
+    public <R extends CommonDistributionConfigRestModel> Class<R> getDistributionRestModelClass();
+
     public boolean hasGlobalConfiguration();
 
+    public <R extends JpaRepository<DatabaseEntity, Long>> R getDistributionRepository();
+
+    public <A extends SimpleConfigActions> A getSimpleConfigActions();
```

Comment:
Rename this to getConfigActions SimpleConfigActions are implied based on the generic type A you have specified.